README.md

```diff
@@ -53,8 +53,26 @@ Currently there is only a draft version of layer renderer for Tiled utilizing `h
 
 ### Manifest FS
 Since we are sane people and don't want 50+MB js file that contains Base64-encoded game assets, we obviously want to load those files separately. Manifest-FS provides ability to load those files from a manifest file.  
-This approach requires some prep-work to get it running, but beats embedding everything in JS.  
-First, you have to generate manifest file with `cherry.fs.ManifestBuilder.create`, `generate` and `initManifest`. Last acts exactly the same as `Res.init*` functions and bakes manifest into the code. First just generates manifest FS in macro call and second one just does convert and optionally saves manifest to your res folder. I trust people here are smart enough to figure out how to utilize those two, so I'll focus on one I use, e.g. `initManifest` method.
+This approach requires some prep-work to get it running, but beats embedding everything in JS.
+
+First, generate a manifest file with `cherry.fs.ManifestBuilder.generate`, like so:
+
+``` haxe
+cherry.fs.ManifestBuilder.generate("res", null, "manifest.json");
+```
```

Comment:
This is incorrect, as it serves no real purpose with the snippet below. `initManifest` embeds manifest file into source code. `generate` is used for more non-trivial system where files may be changed without updating the build, in which case storing manifest in separate file makes more sense, where you manually create MFS instance and pass manifest to it, which is loaded separately. `create` is "light" version of `initManifest`, i.e. it only creates MFS instance, but does not assign it as Res loader, which can be utilized when multiple FS are used.